<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Desain Gambar Denah</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_desain">
        <attribute name="title">
         <string>Desain</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_provinsi">
               <property name="text">
                <string>Provinsi</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="cmb_propinsi"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_kabupaten">
               <property name="text">
                <string>Kabupaten/Kota</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cmb_kabupaten"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_kecamatan">
               <property name="text">
                <string>Kecamatan</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="cmb_kecamatan"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_kelurahan">
               <property name="text">
                <string>Desa/Kelurahan</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QComboBox" name="cmb_desa"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Lihat Data</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="cmb_lihat_data"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="dgv_GambarDenah">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_error">
        <attribute name="title">
         <string>Error Log</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTextEdit" name="error_log"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="chb_Sistem_Koordinat">
       <property name="text">
        <string>Sistem Koordinat TM3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_validasi">
       <property name="text">
        <string>Validasi</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_proses">
       <property name="text">
        <string>Proses</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_batal">
       <property name="text">
        <string>Batal</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_status_l">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_status_r_2">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
